WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -811,41 +810,49 @@ - (void)showDeleteSiteForBlog:(Blog *)blog
 - (void)tableView:(UITableView *)tableView didSelectInAdvancedSectionRow:(NSInteger)row
 {
     switch (row) {
-        case SiteSettingsAdvancedStartOver: {
+        case SiteSettingsAdvancedStartOver:
             [self showStartOverForBlog:self.blog];
-        } break;
-        case SiteSettingsAdvancedDeleteSite: {
+            break;
+
+        case SiteSettingsAdvancedDeleteSite:
             [self showDeleteSiteForBlog:self.blog];
-        } break;
+            break;
     }
 }
 
 - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
 {
+    [tableView deselectSelectedRowWithAnimation:YES];
```

Comment:
Since we add `deselectSelectedRowWithAnimation:` here it will prevent the default `UITableViewController` behavior via `clearsSelectionOnViewWillAppear`. Not a big deal, but general behavior to have a cell stay selected until returning from a detail view.